src/svc.c

```diff
@@ -230,39 +230,13 @@ svc_init(svc_init_params *params)
 	return true;
 }
 
-struct rpc_msg *
-alloc_rpc_msg(void)
-{
-	struct rpc_msg *msg = mem_alloc(sizeof(struct rpc_msg));
-
-	TAILQ_INIT_ENTRY(msg, msg_q);
```

Comment:
I'm worried about the rpc_msg itself.  In client call, iirc, we allocate one, but when bi-directional, we get a pointer to one allocated in svc_recv (or its caller?) and work with it.